net-misc/feather/feather-1.0.1.ebuild

```
@@ -0,0 +1,61 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A free, open-source Monero wallet"
+HOMEPAGE="https://featherwallet.org"
+SRC_URI="https://featherwallet.org/files/releases/linux/${PN}-${PV}-linux.zip -> ${P}.zip"
+
+# Feather is released under the terms of the BSD license, but it vendors
+# code from Monero and Tor too.
+LICENSE="BSD MIT"
+SLOT="0"
+KEYWORDS=""
+IUSE="xmrig +xmrto"
+
+DEPEND="
+	dev-libs/boost:=[nls,threads]
+	dev-libs/libgcrypt:=
+	dev-libs/libsodium:=
+	dev-libs/monero-seed
+	dev-libs/openssl:=
+	>=dev-qt/qtcore-5.15
+	>=dev-qt/qtgui-5.15
+	>=dev-qt/qtnetwork-5.15
+	>=dev-qt/qtsvg-5.15
+	>=dev-qt/qtwebsockets-5.15
+	>=dev-qt/qtwidgets-5.15
+	>=dev-qt/qtxml-5.15
+	media-gfx/qrencode:=
+	net-dns/unbound:=[threads]
+	net-libs/czmq:=
+"
+RDEPEND="
+	${DEPEND}
+	net-vpn/tor
+"
```

No deps are needed at build time now.
```suggestion
RDEPEND="
	dev-libs/boost:=[nls,threads]
	dev-libs/libgcrypt:=
	dev-libs/libsodium:=
	dev-libs/monero-seed
	dev-libs/openssl:=
	>=dev-qt/qtcore-5.15
	>=dev-qt/qtgui-5.15
	>=dev-qt/qtnetwork-5.15
	>=dev-qt/qtsvg-5.15
	>=dev-qt/qtwebsockets-5.15
	>=dev-qt/qtwidgets-5.15
	>=dev-qt/qtxml-5.15
	media-gfx/qrencode:=
	net-dns/unbound:=[threads]
	net-libs/czmq:=
	net-vpn/tor
"
```

---

Actually, it looks like a lot of things are statically linked:

```
$ ldd feather-1.0.1
        linux-vdso.so.1 (0x00007ffe403fd000)
        libudev.so.1 => /lib64/libudev.so.1 (0x00007fb1fb544000)
        libdl.so.2 => /lib64/libdl.so.2 (0x00007fb1fb53f000)
        libX11-xcb.so.1 => /usr/lib64/libX11-xcb.so.1 (0x00007fb1fb53a000)
        libxcb-icccm.so.4 => /usr/lib64/libxcb-icccm.so.4 (0x00007fb1fb533000)
        libxcb-image.so.0 => /usr/lib64/libxcb-image.so.0 (0x00007fb1fb52d000)
        libxcb-shm.so.0 => /usr/lib64/libxcb-shm.so.0 (0x00007fb1fb528000)
        libxcb-keysyms.so.1 => /usr/lib64/libxcb-keysyms.so.1 (0x00007fb1fb521000)
        libxcb-randr.so.0 => /usr/lib64/libxcb-randr.so.0 (0x00007fb1fb50f000)
        libxcb-render-util.so.0 => /usr/lib64/libxcb-render-util.so.0 (0x00007fb1fb509000)
        libxcb-render.so.0 => /usr/lib64/libxcb-render.so.0 (0x00007fb1fb4f9000)
        libxcb-shape.so.0 => /usr/lib64/libxcb-shape.so.0 (0x00007fb1fb4f4000)
        libxcb-sync.so.1 => /usr/lib64/libxcb-sync.so.1 (0x00007fb1fb4ea000)
        libxcb-xfixes.so.0 => /usr/lib64/libxcb-xfixes.so.0 (0x00007fb1fb4de000)
        libxcb-xinerama.so.0 => /usr/lib64/libxcb-xinerama.so.0 (0x00007fb1fb4d9000)
        libxcb-xkb.so.1 => /usr/lib64/libxcb-xkb.so.1 (0x00007fb1fb4ba000)
        libxcb-xinput.so.0 => /usr/lib64/libxcb-xinput.so.0 (0x00007fb1fb495000)
        libxcb.so.1 => /usr/lib64/libxcb.so.1 (0x00007fb1fb46b000)
        libX11.so.6 => /usr/lib64/libX11.so.6 (0x00007fb1fb328000)
        libxkbcommon-x11.so.0 => /usr/lib64/libxkbcommon-x11.so.0 (0x00007fb1fb31b000)
        libz.so.1 => /lib64/libz.so.1 (0x00007fb1fb301000)
        libxkbcommon.so.0 => /usr/lib64/libxkbcommon.so.0 (0x00007fb1fb2ba000)
        libglib-2.0.so.0 => /usr/lib64/libglib-2.0.so.0 (0x00007fb1fb183000)
        libm.so.6 => /lib64/libm.so.6 (0x00007fb1fb0ac000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00007fb1fb0a7000)
        libc.so.6 => /lib64/libc.so.6 (0x00007fb1faeac000)
        /lib64/ld-linux-x86-64.so.2 (0x00007fb1fe6e9000)
        libxcb-util.so.1 => /usr/lib64/libxcb-util.so.1 (0x00007fb1faea4000)
        libXau.so.6 => /usr/lib64/libXau.so.6 (0x00007fb1fae9f000)
        libXdmcp.so.6 => /usr/lib64/libXdmcp.so.6 (0x00007fb1fae97000)
        libpcre.so.1 => /lib64/libpcre.so.1 (0x00007fb1fae1f000)
        libbsd.so.0 => /usr/lib64/libbsd.so.0 (0x00007fb1fae06000)
        libmd.so.0 => /usr/lib64/libmd.so.0 (0x00007fb1fadf9000)
```